src/operator/subgraph/subgraph_property.h

```diff
@@ -221,6 +222,12 @@ class SubgraphProperty {
     return nullptr;
   }
 
+  virtual void PrePartition(const nnvm::Graph& g,
+    const std::vector<std::pair<std::string, std::string>>& options_map) {}
+
+  virtual void PostPartition(const nnvm::Graph& g,
+    const std::vector<std::pair<std::string, std::string>>& options_map) {}
```

Comment:
Dont think we need options_map in the postPartition args list. Any options that need to be saved can be saved in the SubgraphProperty Class.

Reply:
Removed options_map from PostPartition